pkg/restore/actions/csi/volumesnapshot_action.go

```diff
@@ -106,12 +108,20 @@ func (p *volumeSnapshotRestoreItemAction) Execute(
 		return nil, errors.WithStack(err)
 	}
 
-	p.log.Infof(`Returning from VolumeSnapshotRestoreItemAction with 
-		no additionalItems`)
+	// Adding VS's VolumeSnapshotContent
+	additionalItem := velero.ResourceIdentifier{
+		GroupResource: schema.GroupResource{
+			Group:    "snapshot.storage.k8s.io",
+			Resource: "volumesnapshotcontents",
+		},
+		Name: *vsFromBackup.Status.BoundVolumeSnapshotContentName,
```

Comment:
You don't check if the vsFromBackup has Status nor the Status.BoundVolumeSnapshotContentName, which could lead to panic if missing?